frontend/src/metabase/entities/questions.js

```diff
@@ -77,6 +80,17 @@ const Questions = createEntity({
   },
 
   reducer: (state = {}, { type, payload, error }) => {
+    if (type === SOFT_RELOAD_CARD) {
+      const { id } = payload;
+      const verified =
+        payload.moderation_reviews?.find(x => x.most_recent)?.status ===
+        "verified"
+          ? "verified"
+          : null;
```

Comment:
Do we actually need a ternary here? Maybe we should just find the most recent status and add it to the question state if it's not `undefined`? AFAIR `moderated_status` can be something else apart from `verified` and `null`

Reply:
Should we maybe extract this lookup to a function or break it down a little? I feel it can be a bit hard to follow. Maybe something like:

```js
const latestReview = payload.moderation_reviews?.find(x => x.most_recent)
if (state[id] && latestReview) {
  state[id].moderated_status = latestReview.status
}
```

Reply:
@npfitz you would also need to return a new state object without modifying the existing one. Maybe something like this: https://github.com/metabase/metabase/blob/501f4d1a50dbe03d64cdfeee86a87d680eab3d33/frontend/src/metabase/entities/bookmarks.js#L66

Reply:
Awesome! Will do